lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -296,6 +297,26 @@ def fit_power_spectrum(
     solution_params = scaled_model.scale_params(solution_params_rescaled)
     perr = scaled_model.scale_params(perr)
 
+    # Use the least squares method as a starting point for a robust fit
+    if robust:
+        scaled_model = ScaledModel(
+            lambda f, fc, D, *filter_pars: model(f, fc, D, *filter_pars), solution_params
+        )
+        sigma_r = (power_spectrum.power) / math.sqrt(power_spectrum.num_points_per_block)
```

Comment:
Minor point, but sigma makes me think of standard deviations, considering that the Lorentzian doesn't have a defined variance, it seems a bit of an odd choice.

This does raise a question though. The scale of these weights, how did you choose it? Did you use some sort of matching criterion between the distributions?

Reply:
It was more or less heuristically, keeping the error term the same for LSQ and WLE (`(y[i] - y(x[i]))**2 / sigma**2`), knowing that _most_ data points would actually have a calculable sigma. Under extreme pressure, I'd probably claim I matched the order of magnitude of the Mean Absolute Deviation ;)